app/src/main/java/com/android/sample/model/filter/Filter.kt

```diff
@@ -41,19 +32,15 @@ data class FloatRange(var min: Float, var max: Float, var minBorn: Float, var ma
   }
 
   fun update(newMin: Float, newMax: Float) {
```

Comment:
**`update` Method:**  
   Love how you simplified this! Removing unnecessary checks makes it way cleaner. Just double-check edge cases are covered in tests.

Reply:
All the essential tests have been retained. The only changes involved removing edge cases, which are now supported directly by this function, enhancing its usability and coverage.